chapters/executors.rst

```diff
@@ -0,0 +1,113 @@
+
+|
+
+.. image:: /chapters/images/pea_exec_driver.png
+	:width: 60%
+
+|
+
+The problem now is how can they communicate the data they are processing. And this is where we meet the :term:`Driver`.
+
+Drivers
+^^^^^^^^^^^^^^^
+
+The :term:`Driver` are the ones that handle the input and output messages from the :term:`Executor`.
+
+As well as we have different Executors for different tasks, we also need different Drivers for different Execturos, and you can find the `list here <https://docs.jina.ai/chapters/all_driver.html>`_.
+
+Executors in action
+----------------------
+
+:term:`Executors` can be used in several ways in Jina.
+
+Like we said, we have 7 types of Execturos in Jina, so let's see an example with one of them and how we could see it in action.
+
+Run with Docker (docker run)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First Let's use a `Ranker <https://docs.jina.ai/chapters/ranker>`_ just for this example, but this can be used for any other type of Exectuor that you need.
+
+
+.. highlight:: bash
+.. code-block:: bash
+
+    docker run jinahub/pod.ranker.simpleaggregateranker:MODULE_VERSION-JINA_VERSION --port-in 55555 --port-out 55556
+
+Run with Flow API
+^^^^^^^^^^^^^^^^^^
+
+Another way to use them is directly in your Phyton file. Let's use a :term:`Crafter` this time, for example the `Image Normalizer <https://github.com/jina-ai/jina-hub/tree/master/crafters/image/ImageNormalizer>`_
+
+.. highlight:: python
+.. code-block:: python
+
+     from jina.flow import Flow
+
+     f = (Flow().add(name='my_crafter', uses='docker://jinahub/pod.crafter.imagenormalizer:MODULE_VERSION-JINA_VERSION'))
+     # Or use YAML file.
+     #f = (Flow().add(name='my_crafter', uses='imagenormalizer.yml'))
+
+Run with Jina CLI
+^^^^^^^^^^^^^^^^^^
+
+Or directly with Jina CLI. Now let's use an Indexer, the `RedisBDIndexer <https://github.com/jina-ai/jina-hub/tree/master/indexers/keyvalue/RedisDBIndexer>`_
+
+.. highlight:: bash
+.. code-block:: bash
+
+        jina pod --uses docker://jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION
+
+Conclusion
+-----------------
+
+In this guide, we introduced why we need and how to use :term:`Executors` and how they need :term:`Drivers` to communicate. Apart from that, we provided some concrete examples of how to use them. Now that you now what they are and how to use them, you might be wondering how to create them, we have a guide on that for `Executors <https://docs.jina.ai/api/jina.executors.html>`_ and `Drivers <https://docs.jina.ai/api/jina.drivers.html>`_.
+
+What's next
+-----------------
+
+If you still have questions, feel free to `submit an issue <https://github.com/jina-ai/jina/issues>`_ or post a message in our `community slack channel <https://slack.jina.ai>`_ .
+
+To gain a deeper knowledge on the implementation of Jina Ranker, you can find the source code `here <https://github.com/jina-ai/jina/tree/master/jina/executors/rankers>`_.
```

Comment:
the link should be `here <https://github.com/jina-ai/jina/tree/master/jina/executors/>`_.